clients/scheduler/webpack.config.js

```diff
@@ -51,7 +51,7 @@ module.exports = ({ esm, prod }) => {
       rules: [
         {
           test: /\.tsx?$/,
-          exclude: /node_modules/,
+          exclude: /\.\/node_modules/,
```

Comment:
Just making sure this is intentional

Reply:
This is what lets TS actually work when we run it :)